ix-dev/stable/nextcloud/templates/docker-compose.yaml

```diff
@@ -117,8 +117,8 @@
   {% do nc_env.x.append(("OVERWRITEPROTOCOL", "https")) %}
   {% do nc_env.x.append(("TRUSTED_PROXIES", ["127.0.0.1", "192.168.0.0/16", "172.16.0.0/12", "10.0.0.0/8"] | join(" "))) %}
   {% do nc_confs.append(("logformat.conf", use_x_real_ip_in_logs(), "/etc/apache2/conf-enabled/logformat.conf", "")) %}
-  {% if values.nextcloud.host and values.network.nginx.use_different_port %}
-    {% set host.x = "%s:%d"|format(values.nextcloud.host, values.network.nginx.external_port) %}
+  {% if values.nextcloud.host %}
+    {% set host.x = "%s:%d"|format(values.nextcloud.host, values.network.nginx.external_port if values.network.nginx.use_different_port else values.network.web_port) %}
```

Comment:
sadly this change also requires migration. because users that dont have use_different_port checked and have already set OVERWRITEHOST. will receive an error that this variable is already set and wont be able to upgrade.

I'm still not 100% sure what this change is supposed to do, but so far it only sets the OVERWRITEHOST when cert is defined.